src/QueryReflection/RuntimeConfiguration.php

```diff
@@ -53,6 +58,18 @@ public function errorMode(string $mode): self
         return $this;
     }
 
+    /**
+     * Defines default fetch mode.
+     *
+     * @param QueryReflector::FETCH_TYPE* $mode
+     */
+    public function fetchMode(int $mode): self
```

Comment:
I will rename that one to `defaultFetchMode`, as it will only be the fallback for cases no fetch-mode is defined otherwise